.github/ISSUE_TEMPLATE/bug_report_linux.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report (Linux)
+description: File a bug report for Linux.
+labels: ["bug", "triage"]
+projects: ["freedomofpress/dangerzone"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi, and thanks for taking the time to open this bug report. 
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Can you specify the steps you followed, so that we can reproduce?
```

Comment:
Maybe start with "What was the expected behavior, and what was the actual behavior?".